### PR TITLE
Issue 1 create a profile

### PR DESCRIPTION
**Pull request by cbrisk**
image

### CODE REVIEW

**Comment by cbrisk** [comment]
image
image
image
image

**Comment by cbrisk** [comment]
image
image
image
image

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  data.profile.avatarUrl = $form.elements.avatarUrl.value;
+  data.profile.username = $form.elements.username.value;
+  data.profile.fullName = $form.elements.fullName.value;
+  data.profile.location = $form.elements.location.value;
+  data.profile.bio = $form.elements.bio.value;
+  var inputData = JSON.stringify(data);
+  localStorage.setItem('profile', inputData);
```
Comment (codingcodymiller) [inline js/main.js:17]: You should only do this before the page unloads, not every time the form is submitted 👍

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  $form.reset();
+  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+});
+
+/* window.addEventListener('beforeunload', function (event) {
```
Comment (codingcodymiller) [inline js/main.js:22]: I see you have commented code here. If this turned out to be unnecessary, you should remove it 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**